pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+A mechanism is needed to capture the sense of the community and allow it to progress while recognizing that the community is broad, diffuse, and diverse, and not all stakeholders are able to be present for all conversations.
+
+# PHEP Audience
+<a name="phep-audience"></a>
+The typical primary audience for PHEPs are the developers of PyHC packages.
+
+However, other parts of the PyHC community (hereafter "community") may also choose to use the process (particularly for Informational PHEPs) to document expected conventions and to manage complex design coordination problems that require collaboration across multiple projects.
+
+This PHEP makes reference to "PyHC leadership": individuals or groups with statutory authority over the PyHC project. This may be the Principal Investigator or other individuals or groups as appropriately designated in the future, such as a steering committee.
+
+# PHEP Types
+There are three kinds of PHEP:
+
+1. A **Standards** Track PHEP describes a new standard for PyHC packages.
+2. An **Informational** PHEP describes a design issue, or provides general guidelines or information to the community, but does not propose a new standard. Informational PHEPs do not necessarily represent a community recommendation, so users and implementers are free to ignore Informational PHEPs or follow their advice.
+3. A **Process** PHEP describes a process surrounding PyHC, or proposes a change to (or an event in) a process. Process PHEPs are like Standards Track PHEPs but apply to areas other than PyHC packages. They may propose an implementation; they often require community consensus; unlike Informational PHEPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in PyHC development or other community activities. Any meta-PHEP is also considered a Process PHEP.
+
+# PHEP Applicability
+**Standards** Track PHEPs are recommended standards for PyHC packages. Most packages should make a good-faith effort to comply. PyHC core packages are strongly recommended to comply and generally should do so, absent a compelling reason not to. A PHEP which core packages cannot agree to implement likely does not have sufficient community support for approval. Standards track PHEP compliance should be considered as a factor in evaluating current and potential PyHC packages.
+
+**Informational** PHEPs are for the benefit of PyHC developers and users and may represent "best practices". They are not binding.
+
+**Process** PHEPs determine the process by which the PyHC community makes formal decisions and as such bind all participants in PyHC.
+
+For all three kinds, key words in PHEPs must be interpreted according to [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119); the use of `must`, `must not`, `should`, and `should not` are preferred to the alternatives.
+
+# PHEP Workflow
+## PHEP Editors
```

Comment:
On the call, we had discussed the possibility of just having Shawn and Julie serve as editors. After the fact, I thought it probably makes more sense to leave project leadership in a mediation role--that keeps them out of the nuts-and-bolts and provides a place to appeal.